engines/base/src/main/resources/templates/error.html

```diff
@@ -0,0 +1,22 @@
+<html xmlns:th="http://www.thymeleaf.org">
```

Comment:
Missing `<!DOCTYPE html>` at start of file.
Missing a `lang` or `xml:lang` attribute.
Should have a `<head>` tag that contains a `<title>`
```suggestion
<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org" xml:lang="en">
    <head>
        <title>"${title}"</title>
    </head>
```

Also check indentation.

Reply:
Needed to use: <title th:text="${title}"/>